source/_integrations/homematic.markdown

```diff
@@ -440,6 +440,37 @@ action:
     entity_id: lock.leq1234567
 ```
 
+
+#### Integrating HMIP-DLD
+
+There is no avaliable default integration for HMIP Doorlock (HMIP-DLD) in the current pyhomeatic implementation.
+A workaround is to define a template lock in configuration.yaml:
+
+```yaml
+lock:
+  - platform: template
+    name: Basedoor
+    unique_id: basedoor
+    value_template: "{{ states.homematic.ccu2.attributes.base_lock_status }}"
```

Comment:
Use `state_attr(...` <https://www.home-assistant.io/docs/configuration/templating/#states>

Reply:
Adapted state_attr().

Reply:
would be happy to get a hint why the checks are now failing. Compared my syntax again with the documentation and i cant find the problem.

Reply:
When using templates you have to wrap the section with `{% raw %}` / `{% endraw %}`